SpatialGDK/Source/Public/Interop/SpatialTypebindingManager.h

```diff
@@ -0,0 +1,101 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#pragma once
+
+#include "CoreMinimal.h"
+#include "Utils/SchemaDatabase.h"
+
+#include <improbable/c_worker.h>
+
+#include "SpatialTypebindingManager.generated.h"
+
+enum ERPCType
+{
+	RPC_Client = 0,
+	RPC_Server,
+	RPC_CrossServer,
+	RPC_NetMulticast,
+	RPC_Count
```

Comment:
uber nit: I dislike enum count names not standing out for normal enum names - `RPC_COUNT`?